universal-application-tool-0.0.1/app/models/Program.java

```diff
@@ -36,6 +36,9 @@
 
   @Constraints.Required private String description;
 
+  /** Link to external site for this program. */
+  @Constraints.Required private String externalLink;
```

Comment:
question: should this be required? Or is it ok for a program to not have an external link?

Reply:
I could go either way but I prefer required. Let me know if you think I should change it.

Reply:
Sounds good to me!